src/main/java/com/mongodb/hibernate/jdbc/MongoDatabaseMetaData.java

```diff
@@ -90,6 +91,12 @@ public String getSQLKeywords() {
         return "";
     }
 
+    /** @return {@value #SCHEMA_TERM}. */
+    @Override
+    public String getSchemaTerm() {
+        return SCHEMA_TERM;
+    }
+
```

Comment:
I chose to say that a MongoDB database is a JDBC schema (as opposed to saying that it's a JDBC catalog), because semantically [MongoDB database](https://www.mongodb.com/docs/manual/reference/glossary/#:~:text=Center%20Awareness.-,database,in%20the%20file%20system.%20One%20MongoDB%20server%20typically%20has%20multiple%20databases.,-database%20command) is most similar to [PostgreSQL schema](https://www.postgresql.org/docs/current/ddl-schemas.html), which [their JDBC driver uses as the JDBC schema](https://github.com/pgjdbc/pgjdbc/blob/15596302e6ab9ccdefe25c9bd91284d5f232ee27/pgjdbc/src/main/java/org/postgresql/jdbc/PgDatabaseMetaData.java#L713).

Reply:
it might be right. But as a general rule, we only implemented JDBC methods Hibernate really called and we decided to support, right? It seems confusing why we add this method only we could call it. The discussion about the mapping seems irrelevant in this PR. We could provide a database without going through JDBC schema, right?

Reply:
We agreed not to interpret the MongoDB database as JDBC schema, at least for now.

Done in 3bb8ee965f06d51bdc5b8fd333fa346363017c4b.